### DISCUSSION TITLE
Why custom DT multipliers are unranked?

### DISCUSSION THREAD

**Discussion by okolenmi** [ppy/osu#30550] [Q&A] [answered]
I just want to know a reason of this. Nothing offensive, just my thoughts:

Possibility of several DT scores on the same map:
- System can select the best DT score among all DT scores to the Best Performance table. 

"Players can set a score on a comfortable bpm to get more pp":
- It's a problem of pp system, if you think it can be an advantage.  Also it's already a game where people can get pp for their best osu skills (streams, reaction speed, aim, etc.), Imho it will be nice to add more pp for very dense (excluding streams) 300+ bpm maps as they are obviously underestimated. It's easy to press 150 bpm spacing streams, but it's another thing when every next circle changes direction.

**Comment by ziv-vy** [comment]
Because of exploits

**Comment by peppy** [comment, marked as answer]
Because we're waiting for the throughput to make sure they are in a good place, balance-wise. Once the flood gates are opened, it will generate a lot of extra work for us if we get things wrong. Please be patient.
